writerseeker.go

```diff
@@ -58,5 +68,5 @@ func (ws *WriterSeeker) Close() error {
 
 // BytesReader returns a *bytes.Reader. Use it when you need a reader that implements the io.ReadSeeker interface
 func (ws *WriterSeeker) BytesReader() *bytes.Reader {
```

Comment:
It's unrelated to this change, but I'm not sure that this method is needed (although it might be hard to remove now, since people may be using it). Could `Reader` instead return an `io.ReadSeeker`?